ambient.d.ts

```diff
@@ -246,18 +246,38 @@ declare module '@mojaloop/central-services-shared' {
       TP_CONSENT_PUT_ERROR: string;
     };
   }
+
+  enum EventTypeEnum {
+    AUTHORIZATION = 'AUTHORIZATION',
+    NOTIFICATION = 'NOTIFICATION',
+    TRANSACTION_REQUEST = 'TRANSACTION_REQUEST',
+  }
+
+  enum EventActionEnum {
```

Comment:
why only partial Action is specified: 
https://github.com/mojaloop/central-services-shared/blob/master/src/enums/events.js#L52
there is a difference where in source code as enum values lowercase is used, here uppercase